torch/quantization/_quantize_script.py

```diff
@@ -64,6 +64,16 @@ def __setstate__(self, state):
     linear_packed_params = torch.jit.script(torch.nn.quantized.modules.linear.LinearPackedParams())._c
     conv_packed_params = torch.jit.script(ConvPackedParams())._c
 
+def _preprocess_qconfig_dict_helper(module, qconfig_dict, prefix=''):
+    for name, child in module.named_children():
+        module_prefix = prefix + '.' + name if prefix else name
+        # Existing string names in dict take precedence over type names.
+        if type(child) in qconfig_dict.keys() and module_prefix not in qconfig_dict.keys():
+            qconfig_dict[module_prefix] = qconfig_dict[type(child)]
+        else:
+            qconfig_dict.update(_preprocess_qconfig_dict_helper(child, qconfig_dict, module_prefix))
+    return qconfig_dict
```

Comment:
also I think we should make this function a inplace function

Reply:
also please remember to change _preprocess_qconfig_dict_helper to `_preprocess_qconfig_dict_helper_` and qconfig should be the first argument of this function because that is the primary object we are preprocessing.